src/pdfboxing/info.clj

```diff
@@ -24,6 +24,21 @@
   [pdfdoc]
   (with-open [doc (common/load-pdf pdfdoc)]
     (let [info (.getDocumentInformation doc)
-          info-fields ["title" "author" "subject" "keywords" "creator" "producer"]]
+          info-fields ["title" "author" "subject" "keywords" "creator" "producer" "trapped" "metadata-keys" "creation-date" "modification-date"]]
```

Comment:
The indentation of the above line is really so that it doesn't stand out from this line, which I think would be more legible if indented as such:

```clojure
    (let [info (.getDocumentInformation doc)
          info-fields ["title" "author" "subject" "keywords"
                       "creator" "producer" "trapped" "metadata-keys"
                       "creation-date" "modification-date"]]
```